buildspec-toolkit.yml

```diff
@@ -3,7 +3,7 @@ version: 0.2
 env:
   variables:
     FRAMEWORK_VERSION: '1.6.0'
-    EIA_FRAMEWORK_VERSION: '1.4.1'
+    EIA_FRAMEWORK_VERSION: '1.5.1'
```

Comment:
Not sure, But this change in buildspec-toolkit.yml is needed after the the image is ready.
Let refer @saimidu on this ?

Reply:
@TusharKanekiDey is right.

The toolkit integration tests test the "sagemaker-mxnet-inference" package on a pre-existing DLC image of the given `FRAMEWORK_VERSION`. This pre-existing DLC image is pulled from the publicly accessible ECR repository to which images are published upon a DLC release.

There is no MXNet 1.5.1 EI DLC that has already been released yet, until this PR is merged and used for a release.

Therefore, changing the framework version on the buildspec-toolkit workflow will force this workflow to fail, until the first release for MXNet 1.5.1 EI is complete.

Reply:
Got it. Will remove this